docs/UserGuide.md

```diff
@@ -374,3 +374,6 @@ It is also possible that another user has overriden the authorship of your lines
 
 #### Q: I am able to run RepoSense on my repository on a Linux/Mac OS, but it fails on a Windows OS?
 **A:** It is possible you may have some file names with [special characters](https://docs.microsoft.com/en-us/windows/desktop/FileIO/naming-a-file#naming-conventions) in them, which is disallowed in Windows OS. As such, *RepoSense* is unable to fully clone your repository, thus failing the analysis.
+
+#### Q: Some files are not captured by RepoSense!
+**A:** The files may be [binary files](https://www.webopedia.com/TERM/B/binary_file.html). RepoSense does not analyze binary files. Common binary files include images (`.jpg`, `.png`), applications (`.exe`), zip files (`.zip`, `.rar`) and certain document types (`.docx`, `.pptx`).
```

Comment:
Let's use Wikipedia instead? https://en.wikipedia.org/wiki/Binary_file
The link contains ads that we have no control on.

Reply:
Ok noted. Wasn't aware of this as I'm running ad blockers